### DISCUSSION TITLE
Is it possible to install Causalml without torch?

### DISCUSSION THREAD

**Discussion by minh-nguyen-dh** [uber/causalml#480] [Q&A] [answered]
Hi!

I am trying to package my code. However the size is too big if I install it along with `torch` as a dependency. Since my code only uses tree models, would there be a way to specify `causalml` in requirements.txt without `torch`?

Thanks for reading.

**Comment by 00helloworld** [comment, marked as answer]
Hi, if you don't need `torch`, you can install `causalml` with this code:
`pip install causalml --no-deps torch`
